lib/perl/centreon/script/centcore.pm

```diff
@@ -558,8 +581,8 @@ sub sendExportFile($){
         return;
     }
 
-    unless ( -e $self->{centreonDir}  . "filesGeneration/export/".$id) {
-        $self->{logger}->writeLogInfo("Export directory is empty for poller $self->{centreonDir}filesGeneration/export/$id.");
+    if ( -e $self->{centreonDir}  . "/filesGeneration/export/".$id) {
```

Comment:
execution will stop if export with $id exists in that case

Reply:
probably those changes were because the state was branched out from master earlier and then they were applied to script